code/game/turfs/simulated/floor_glass.dm

```diff
@@ -28,7 +28,7 @@
 	icon_state = "[((x + y) ^ ~(x * y) + z) % 25]"
 	if(!floor_overlays[glass_state])
 		var/image/floor_overlay = image('icons/turf/overlays.dmi', glass_state)
-		floor_overlay.plane = TURF_PLANE
+		floor_overlay.plane = GLASSTILE_PLANE
```

Comment:
since you're adding an image as an overlay it might be good practice to start using the `relative_plane(GLASSTILE_PLANE)` macro defined in planes+layers.dm. Same for the one below.

Reply:
Fixed, still works as intended.